src/game_api/level_api.hpp

```diff
@@ -458,9 +460,17 @@ struct LevelGenSystem
     std::uint32_t spawn_room_y;
     float spawn_x;
     float spawn_y;
-    DoorCoords* exit_doors_locations;
-    void* unknown37;
-    void* unknown38;
+    union
+    {
+        std::vector<ImVec2> exit_doors;
```

Comment:
Oh, I see it's for this. I'd prefer if we don't depend on imgui on the game types, that will make it more annoying later to split just the types and functionality (for pulling out types into their own headers as optional additions to the .dll interface).

Reply:
I used this just cuz we already have this in API, exposed etc.

Reply:
Well I will have to remove it later either way.

Reply:
ok, then i will make some quick XY struct

Reply:
Works for me  :)
And sorry, I didn't mean to be rude just there. It made perfect sense what you did without the context.